src/components/strategies/overlapping/OverlappingStrategyGraph.tsx

```diff
@@ -520,15 +537,21 @@ export const OverlappingStrategyGraph: FC<Props> = (props) => {
   return (
     <figure className="relative">
       <figcaption className="text-10 absolute inset-x-0 top-0 flex items-center justify-center gap-4 p-16 text-white/60">
-        {props.disabled || props.externalPrice === +props.marketPrice ? (
+        {isPriceAvailable && (
           <>
-            <span>Market price provided by CoinGecko</span>
-            <IconCoinGecko className="size-8" />
+            {props.disabled ||
+            (!!props.marketPrice &&
+              props.externalPrice === +props.marketPrice) ? (
```

Comment:
Maybe create a variable with a good name for that. It's difficult to read

Reply:
Renamed